queries.sql

```diff
@@ -89,9 +113,14 @@ SELECT billing_state, COUNT(*), SUM(total) FROM invoices WHERE billing_country =
 SELECT billing_state, COUNT(*), SUM(total), AVG(total) FROM invoices WHERE billing_country = 'USA' AND billing_state = 'CA' GROUP BY billing_state;
 
 -- The count, total, and average of invoice totals, grouped by state, ordered by average invoice total from highest-to-lowest
+SELECT billing_state, COUNT(*), SUM(total), AVG(totals) FROM invoices WHERE billing_country = 'USA' GROUP BY billing_state ORDER BY AVG(total) DESC;
 
 -- A list of the top 5 countries by number of invoices
+SELECT billing_country, COUNT(*) FROM invoices GROUP BY billing_country ORDER BY COUNT(*) DESC LIMIT 5;
```

Comment:
This query is correct. This is what you'd see in the Postgres shell: https://share.getcloudapp.com/jkuQZBPK

Look at the written examples and be sensitive to the order of the clauses. Each keyword like `SELECT`, `FROM`, `WHERE`, `ORDER BY`, `LIMIT`, etc. signals the start of a clause.